src/controllers/InfoController.php

```diff
@@ -16,7 +16,9 @@ class InfoController extends Controller
 {
     public function showAction($id)
     {
+        $image_baseURL='https://image.tmdb.org/t/p/w1280/';
```

Comment:
Please consider adding a space before and after the `=` to be consistent with styling on lines 20-22.